SofaKernel/modules/SofaBaseLinearSolver/CompressedRowSparseMatrix.h

```diff
@@ -174,6 +174,11 @@ public :
     {
     }
 
+    ~CompressedRowSparseMatrix()
```

Comment:
you are right... but It is maybe useful to have a destructor for CompressedRowSparseMatrix that does a clear() before destruction.